Python/Tests/TestAdapterTests/TestExecutorTests.cs

```diff
@@ -39,6 +39,11 @@ public abstract class TestExecutorTests {
 
         protected abstract PythonVersion Version { get; }
 
+        [ClassCleanup]
+        public static void Cleanup() {
+            TestEnvironment.Clear();
```

Comment:
Class is all tests related to python version x.  This clears all environments created by tests in that class